docs/modules/ROOT/pages/redis/create.adoc

```diff
@@ -0,0 +1,76 @@
+= Create a Redis service
+
+AppCat allows DevOps engineers to quickly provision a redis instance.
+
+== Create a redis service
+
+include::user:ROOT:partial$login-in-terminal.adoc[]
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to create an Object Storage bucket
+[source,yaml]
+----
+apiVersion: appcat.vshn.io/v1
+kind: ExoscaleRedis
+metadata:
+  name: my-redis-example <1>
+  namespace: my-namespace <2>
+spec:
+  parameters:
+    service:
+      redisSettings:
+        maxmemory_policy: noeviction <3>
+      zone: ch-dk-2 <4>
+    size:
+      plan: hobbyist-2 <5>
+  writeConnectionSecretToRef:
+    name: redis-creds <6>
+----
+<1> Instance name: *don't just copy/paste this code, remember to choose your own name here!*
+<2> The namespace where the object will be created.
+<3> Specify custom redis settings
+<4> The exoscale zone to use
```

Comment:
```suggestion
<4> The Exoscale zone to use
```

Afaik there is should be a list linked... We have one here https://docs.appuio.cloud/appcat/object-storage/reference.html, but it's not ideal and probably should be global to the appcat docs

Reply:
True that. It's also not in the CRD if I remember correctly. Might need a clean up.